### ISSUE TITLE
Edit window attributes at runtime

### ISSUE THREAD

**Issue by ryanisaacg** [ryanisaacg/quicksilver#230]
winit has recently added support for fullscreen, window resizing, etc. at runtime on the major desktop platforms. This would allow in-game options menus, which are always nice to have.

**Comment by ryanisaacg** [comment]
Blocked waiting on #220

**Comment by beatgammit** [comment]
Also resize events triggered by the user (if it's not currently an option, I'm pretty new to this library).
